dir.props

```diff
@@ -1,10 +1,17 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition=" Exists('obj/Microsoft.DotNet.Cli.BuildInfo.props') "
```

Comment:
What should happen? Then VS just won't have those properties available right? Should I remove the condition and always import? Making it an error if somehow the generation of that file failed?